### PR TITLE
Fix docs example in relationships

### PR DESCRIPTION
**Pull request by me-ydv-5**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.5.5] - 2020-10-23
+
+### Fixed
+- Incorrect example for creating relationships
```
Comment (hhenriques) [inline CHANGELOG.md:20]: Given that this is a docs change, I don't think it should be it in the change log.

File: cognite/client/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 from cognite.client._cognite_client import CogniteClient
 
-__version__ = "2.5.4"
+__version__ = "2.5.5"
```
Comment (hhenriques) [inline cognite/client/__init__.py:3]: We're on making a change to docs right? If so, we shouldn't bump

**Review by hhenriques — CHANGES_REQUESTED** [review]

File: cognite/client/__init__.py (hunk 2 above)
Comment (me-ydv-5) [reply cognite/client/__init__.py:3]: That makes my work easier. Thanks!

**Review by hhenriques — APPROVED** [review]
lgtm

**Pull request merged**